nototools/unittests/font_tests.py

```diff
@@ -561,6 +561,59 @@ def test_height_of_lowercase_o(self):
                 self.assertEqual(o_height, n_height)
 
 
+class TestGlyphBounds(FontTest):
+    """Tests that certain glyphs do or don't go outside of given bounds.
+
+    Glyphs must be designated with two class attribute lists "should_exceed" and
+    "should_not_exceed", both containing tuples each with the following four
+    elements:
+     - names: a list containing glyph names as strings.
+     - bounds: a list (xmin, ymin, xmax, ymax), values can be None.
+     - multiplier: a value which, when multiplied by a font's weight class
+         value's difference from 400 (regular), gives a bound adjustment for
+         that font.
```

Comment:
positive values adjust the bounds rectangle outward? (xmin moves left, xmax moves right...)